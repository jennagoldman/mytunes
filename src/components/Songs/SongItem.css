.songDiv {
  width: 300px;
}