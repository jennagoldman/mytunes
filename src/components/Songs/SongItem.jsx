import React from 'react';
import PropTypes from 'prop-types';
import styles from './SongItem.css';

const SongItem = ({ title }) => (
  <div className={styles.songDiv}>
    <h3>{title}</h3>
  </div>
);

SongItem.propTypes = {
  title: PropTypes.string.isRequired
};


export default SongItem;
