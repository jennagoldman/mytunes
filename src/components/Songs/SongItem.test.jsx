import React from 'react';
import { shallow } from 'enzyme';
import SongItem from './SongItem.jsx';

describe('SongItem', () => {
  it('matches a snapshot', () => {
    const wrapper = shallow(<SongItem title={'Out of the Blue'} />);
    expect(wrapper).toMatchSnapshot();
  });
});
