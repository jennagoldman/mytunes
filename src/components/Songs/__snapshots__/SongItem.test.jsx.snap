// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongItem matches a snapshot 1`] = `
<div
  className="songDiv"
>
  <h3>
    Out of the Blue
  </h3>
</div>
`;
